subsys/pm/device.c

```diff
@@ -149,3 +149,39 @@ int pm_device_state_get(const struct device *dev,
 	return dev->pm_control(dev, PM_DEVICE_STATE_GET,
 			       device_power_state);
 }
+
+int pm_device_any_busy_check(void)
+{
+	const struct device *dev = __device_start;
+
+	while (dev < __device_end) {
+		if (atomic_test_bit(&dev->pm->atomic_flags,
+				    PM_DEVICE_ATOMIC_FLAGS_BUSY_BIT)) {
+			return -EBUSY;
+		}
+		++dev;
+	}
+
+	return 0;
+}
+
+int pm_device_busy_check(const struct device *dev)
```

Comment:
would be better and more aligned with the rest of the API if it was `bool pm_device_is_busy()`

Reply:
agreed, adjusted it (also the `any` variant). I have also updated release notes with the changes.

Reply:
well, I've checked it again: the functions return `int`, not `bool`, so I'm not sure about the change in that case. I'd only use `_is_` pattern for boolean checks. Adding DNM until this point is clarified.

Reply:
Missed that suggestion included move to `bool`, so it should be ready now.